src/Integration/LocalServices/IProjectSystemHelper.cs

```diff
@@ -32,46 +31,6 @@ namespace SonarLint.VisualStudio.Integration
     /// </summary>
     public interface IProjectSystemHelper : ILocalService, IVsHierarchyLocator
     {
-        /// <summary>
-        /// Returns the current active solution
-        /// </summary>
-        Solution2 GetCurrentActiveSolution();
```

Comment:
Low-risk cleanup - remove methods from the interface that were not being called from any product code.